src/Wizard/Wizard.jsx

```diff
@@ -276,6 +280,7 @@ Wizard.propTypes = {
       PropTypes.func,
       PropTypes.instanceOf(React.Component),
     ]).isRequired,
+    nextButtonValue: PropTypes.oneOfType([PropTypes.string, PropTypes.node, PropTypes.func]),
```

Comment:
Should the `nextButtonValue` prop on line 291 (top-level prop on `Wizard`) also accept a function?

Reply:
Great question! We were thinking about that, but we couldn't think of a use-case where globally changing the button values would be helpful. That being said, if you think it would be best to keep it consistent then let me know!